UI/Marker/GenericPlacemark.gd

```diff
@@ -6,4 +6,5 @@ signal placemark_removed()
 func _gui_input(event):
 	if event is InputEventMouseButton and (event as InputEventMouseButton).button_mask == BUTTON_RIGHT:
 		emit_signal("placemark_removed")
+		SelectedListManager._clear_list()
```

Comment:
Should the entire list be cleared when the placemark is removed? Wouldnt it be better if only this element was removed from the selection list?

Reply:
In a way, having the placemarks react to the higlight signal would have it behave as if the selection didnt contain the removed element, without having to remove it directly